spec/features/user_adds_coverage_to_a_course_spec.rb

```diff
@@ -22,9 +22,59 @@
     end
   end
 
+  scenario "user adds additional coverage to a course
```

Comment:
the feature description already says something like "user adds coverage to a course" so we don't need to repeat that in the scenario. We could say something like:  `scenario "via unmatched outcome link"` or something like that.